cmake/CMakeLists.txt

```diff
@@ -912,6 +912,8 @@ if (onnxruntime_USE_CUDA)
   set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Xcudafe \"--diag_suppress=expr_has_no_effect\"")
 endif()
 
+# TODO: This is for when TensorRT was not a shared library
+if(false)
```

Comment:
Let's address this now or not to put any change here.

Reply:
Ok, I removed the block and told Steven about it to see if we still want to delay load any libraries now that it's a shared provider.

Reply:
Steven agrees we don't need the delay load stuff, and I moved the other things into the shared library already, so I can delete this now.